docs/src/networking/private-cluster.md

```diff
@@ -1 +1,88 @@
 # Using Private clusters
+
+> Note: This section has to be used only if the CAPOCI manages the workload cluster VCN. If externally managed VCN is
+> used, this section is not applicable.
+
+## Example Spec for Private Cluster
+
+CAPOCI supports Private clusters where the Kubernetes API Server Endpoint is a private IP Address
+and is accessible only within the VCN or peered VCNs. In order to use private clusters, the Control Plane
+Endpoint Subnet has to be marked as private. An example spec is given below.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OCICluster
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: "${CLUSTER_NAME}"
+  name: "${CLUSTER_NAME}"
+spec:
+  compartmentId: "${OCI_COMPARTMENT_ID}"
+  networkSpec:
+    vcn:
+      subnets:
+        - cidr: 10.1.0.8/29
+          name: control-plane-endpoint
+          role: control-plane-endpoint
+          type: private
+        - cidr: 10.1.0.0/29
+          name: control-plane
+          role: control-plane
+          type: private
+        - cidr: 10.1.0.32/27
+          name: service-lb
+          role: service-lb
+          type: public
+        - cidr: 10.1.64.0/20
+          name: worker
+          role: worker
+          type: private
+```
+
+## Example spec for VCN Peering using Dynamic Routing Gateway(Local)
+
+While using Private Clusters, the management cluster needs to talk to the workload cluster. If the
+management cluster and workload cluster are in separate VCN, the VCN peering can be used to connect the management
+and workload cluster VCNS. CAPOCI supports peering of the workload cluster VCN with another VCN in the same region using
+[Dynamic Routing Gateway][drg].
+
+In case of Local VCN peering, a DRG OCID has to be provided and CAPOCI will attach the workload cluster VCN to the
+provided DRG. The recommendation is to attach the management cluster VCN also to the same DRG so that the VCNs are
+peered to each other. More details about Local VCN Peering using DRG can be found [here][drg-local].
+
+An example template for this cluster-template-local-vcn-peering.yaml can be found in the Assets section under the
```

Comment:
```suggestion
An example template for this `templates/cluster-template-local-vcn-peering.yaml` can be found in the Assets section under the
```

Since the file is called out? 🤷‍♂️

Reply:
done